GooglePayment/src/main/java/com/braintreepayments/api/GooglePayment.java

```diff
@@ -77,6 +78,24 @@
      */
     public static void isReadyToPay(final BraintreeFragment fragment,
                                     final BraintreeResponseListener<Boolean> listener) {
+        isReadyToPay(fragment, null, listener);
+    }
+
+    /**
+     * Before starting the Google Payments flow, use
+     * {@link #isReadyToPay(BraintreeFragment, ReadyForGooglePayRequest, BraintreeResponseListener)} to check whether the
+     * Google Payment API is supported and set up on the device. When the listener is called with
+     * {@code true}, show the Google Payments button. When it is called with {@code false}, display other
+     * checkout options.
+     *
+     * @param fragment {@link BraintreeFragment}
+     * @param request {@link ReadyForGooglePayRequest}
+     * @param listener Instance of {@link BraintreeResponseListener<Boolean>} to receive the
+     *                 isReadyToPay response.
+     */
+    public static void isReadyToPay(final BraintreeFragment fragment,
```

Comment:
Do we prefer merchants call this function? Or the `isReadyToPay` defined on line 78?